utils/clients/client.go

```diff
@@ -313,7 +313,7 @@ func (c *SimpleHTTPClient) Do(ctx context.Context, req *http.Request, v interfac
 				Body:            rb,
 			}
 
-			return resp, NewHTTPError(err, req.URL.String(), "response", resp.StatusCode, errorData)
+			return resp, NewHTTPError(err, req.URL.String(), err.Error(), resp.StatusCode, errorData)
```

Comment:
Not sure exactly what thinking here was originally (maybe a bug?), but I needed to change this in order to pass the error message to the coingeckoAsset client so it could ignore JSON decoding errors (they are expected since images are not JSON).

Previously the error message was just "response".